projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -145,7 +145,7 @@ mail.smtp.host=smtp.vmware.com
 datajobs.deployment.builder.imagePullPolicy=IfNotPresent
 
 #Settings for Kerberos authorization provider.
-datajobs.security.kerberos.kerberosPrincipal=""
-datajobs.security.kerberos.keytabFileLocation=
-datajobs.security.kerberos.krb5ConfigLocation=
-datajobs.security.kerberos.enabled=false
+datajobs.security.kerberos.kerberosPrincipal=${SECURITY_KERBEROS_KERBEROS_PRINCIPAL:""}
+datajobs.security.kerberos.keytabFileLocation=${SECURITY_KERBEROS_KEYTAB_FILE_LOCATION:}
+datajobs.security.kerberos.krb5ConfigLocation=${SECURITY_KERBEROS_KRB5_CONFIG_LOCATION:}
+datajobs.security.kerberos.enabled=${SECURITY_KERBEROS_ENABLED:false}
```

Comment:
should enabled be enlisted, then the rest of the configs - since enabled is a prerequisite?